client/src/Movies/Movie.js

```diff
@@ -12,7 +12,7 @@ export default class Movie extends Component {
   componentDidMount() {
     // change this line to grab the id passed on the URL
     const id = 1;
```

Comment:
`const id = 1` is unused now. Either remove this line, or do something like:
`const { id } = this.match.params;` and then pass in `id` as was given in the boilerplate. Of course, if you don't always know that you'll have an id and you want that default value, you could do:
`const id = this.match.params.id || 1;`